src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -266,6 +266,55 @@ Intercycle Dependencies
 * ``baz[^]``: the task ``baz`` from the initial cycle point.
 
 
+.. _tutorial-clock-triggers:
+
+Clock Triggers
+--------------
+
+.. ifnotslides::
+
+   In Cylc, :term:`cycle points <cycle point>` are just task labels. Tasks are
+   triggered when their dependencies are met, regardless of cycle point.
+   But *clock triggers* can be used to force tasks to wait for a particular
+   real time, relative to their cycle point, before running.
+   This is necessary for certain operational and monitoring systems, e.g. for
+   tasks that process real-time data.
+
+   For example in the following workflow the cycle ``2050-01-01T12Z`` will wait
+   until 12:00 on the 1st of January 2050 before running:
```

Comment:
Note, the cycle doesn't wait, the task does.

This isn't an important distinction in this example (only one task) but is where multiple parentless tasks are present, such as the tutorial workflow.

Reply:
```suggestion
   For example in the following workflow the tasks at cycle 
   ``2050-01-01T12Z`` will wait
   until 12:00 on the 1st of January 2050 before running:
```

Reply:
```suggestion
   For example, in the following workflow, the task ``do_this_on_or_after_noon``
   will wait until 12:00 every day before running (starting on January 1st 2050):
```